charts/dex/values.yaml

```diff
@@ -208,3 +208,10 @@ tolerations: []
 # -- [Affinity](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity) configuration.
 # See the [API reference](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#scheduling) for details.
 affinity: {}
+
+# -- [Strategy](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy) deployment configuration.
+# See the [API reference](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy) for details.
+strategy: {}
```

Comment:
It's a bit weird, but I did some research and apparently it's an accepted practice to call this just strategy instead of something like deploymentStrategy.

It's fine for now, but we might want to revisit this in the future.

Reply:
Agree